compiler/rustc_target/src/spec/mips64_openwrt_linux_musl.rs

```diff
@@ -8,7 +8,6 @@ pub fn target() -> Target {
     base.cpu = "mips64r2".to_string();
     base.features = "+mips64r2,+soft-float".to_string();
     base.max_atomic_width = Some(64);
-    base.crt_static_default = false;
```

Comment:
The default for musl will likely change to crt_static_default=false in the future.

Reply:
Currently in https://github.com/rust-lang/rust/blob/master/compiler/rustc_target/src/spec/linux_musl_base.rs#L13 as `true` and with my confusion about what should be where, I wanted to not override the MUSL base.  

I can put it back in though

Reply:
And, it's gone! :D Thanks again @bjorn3